packages/lesswrong/components/review/FrontpageReviewWidget.tsx

```diff
@@ -359,12 +362,18 @@ const FrontpageReviewWidget = ({classes, showFrontpageItems=true}: {classes: Cla
             itemsPerPage: 10
            }}
           >       
-            {eligibleToNominate(currentUser) &&
+            <div>
+              {/* If there's less than 24 hours remaining, show the remaining time */}
+              {voteEndDate.diff(new Date()) < (24 * 60 * 60 * 1000) && <span className={classes.timeRemaining}>
+                {voteEndDate.fromNow()} remaining
```

Comment:
If you never merged (I don't remember and it's too late in the evening for me to be checking) the revert add a day PR, then this does not do what you think it does. It's a date, not a datetime, in its original envisioning.